packages/blade/src/components/ChatMessage/_decision.md

```diff
@@ -0,0 +1,85 @@
+# ChatMessage Decisions
+
+- [Design](#design)
+- [ChatMessage Component](#ChatMessage-component)
+  - [ChatMessage API](#ChatMessage-api)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=100413-32686&t=n9A7LztwEkIsly3v-0)
+
+## ChatMessage Component
```

Comment:
We usually keep Usage section first to give idea of the API and then add props table and then more examples of usage